osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -40,7 +41,7 @@ public class DrawableSlider : DrawableOsuHitObject, IDrawableHitObjectWithProxie
         private Container<DrawableSliderTail> tailContainer;
         private Container<DrawableSliderTick> tickContainer;
         private Container<DrawableSliderRepeat> repeatContainer;
-        private Container<PausableSkinnableSound> samplesContainer;
+        private PausableSkinnableSound slidingSample;
```

Comment:
didn't this just become a container last week? ;)

Reply:
It did yeah, but it doesn't need to be disposed or removed anymore.